tests/cpp/common/test_json.cc

```diff
@@ -567,6 +566,11 @@ TEST(Json, DISABLED_RoundTripExhaustive) {
     } else {
       EXPECT_EQ(get<Number const>(loaded), f);
     }
+  };
+#pragma omp parallel for schedule(static)
+  for (uint32_t i = 0; i < std::numeric_limits<uint32_t>::max(); ++i) {
```

Comment:
This is going to exclude the max value `i < std::numeric_limits<uint32_t>::max()`. Let's make the loop variable `i` 64-bit integer.

Reply:
Oops never mind, didn't see the last line.